lib/services/api/auth/params/auth_reset_password_params.dart

```diff
@@ -0,0 +1,32 @@
+part of '../auth_api_service.dart';
+
+class AuthResetPasswordParams extends ApiParams {
+  factory AuthResetPasswordParams({
+    required String email,
+    String? clientId,
+    String? clientSecret,
+  }) {
+    return AuthResetPasswordParams._(
+      email: email,
+      clientId: clientId ?? Configs.app.api.clientId,
+      clientSecret: clientSecret ?? Configs.app.api.clientSecret,
```

Comment:
the clientId & clientSecret is unchanged for the whole app session, wouldn't it be better to configure it at the higher level as you did with token?